packages/client/modules/meeting/components/ReflectTemplateListTeam.tsx

```diff
@@ -38,13 +38,30 @@ interface Props {
   activeTemplateId: string
   showPublicTemplates: () => void
   teamId: string
-  teamTemplates: ReflectTemplateListTeam_teamTemplates
+  settingsRef: ReflectTemplateListTeam_settings$key
 }
 
 const ReflectTemplateListTeam = (props: Props) => {
-  const {isActive, activeTemplateId, showPublicTemplates, teamId, teamTemplates} = props
+  const {isActive, activeTemplateId, showPublicTemplates, teamId, settingsRef} = props
+  const settings = useFragment(
+    graphql`
+      fragment ReflectTemplateListTeam_settings on RetrospectiveMeetingSettings {
+        templateSearchQuery
+        teamTemplates {
+          ...ReflectTemplateItem_template
+          id
+          name
+        }
+      }
+    `,
+    settingsRef
+  )
+  const {teamTemplates, templateSearchQuery} = settings
   const edges = teamTemplates.map((t) => ({node: {id: t.id}})) as readonly {node: {id: string}}[]
   useActiveTopTemplate(edges, activeTemplateId, teamId, isActive, 'retrospective')
+  const filteredTemplates = teamTemplates.filter(({name}) =>
```

Comment:
+1 curious, could you reuse the following hook? https://github.com/ParabolInc/parabol/blob/c3a29b0323049581c88e4a3dc5313b94ca8876da/packages/client/hooks/useFilteredItems.ts